pkg/network/tcp_peer.go

```diff
@@ -45,6 +72,55 @@ func (p *TCPPeer) WriteMsg(msg *Message) error {
 	}
 }
 
+// Handshaked returns status of the handshake, whether it's completed or not
```

Comment:
nit: dots in the end of comments

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
I will start putting dots in the end of sentences.